libnmstate/schemas/operational-state.yaml

```diff
@@ -287,6 +287,23 @@ definitions:
                     type: string
                   access-tag:
                     type: string
+                link-aggregation:
+                  type: object
+                  required:
+                  - mode
+                  - slaves
+                  properties:
+                    mode:
+                      type: string
+                      enum:
+                      - balance-slb
```

Comment:
Do we want to control this at the schema level? It will make it hard to expend the list in the future.
Adding an interface type is one thing, but limiting a link aggregation mode of an ovs port is something else.

Reply:
I don't see how that is an issue, but I'm fine making it more free.